Source/Scene/Model.js

```diff
@@ -154,8 +154,8 @@ define([
         FAILED : 3
     };
 
-    // GLTF_SPEC: Figure out correct mime types (https://github.com/KhronosGroup/glTF/issues/412)
-    var defaultModelAccept = 'model/vnd.gltf.binary,model/vnd.gltf+json,model/gltf.binary,model/gltf+json;q=0.8,application/json;q=0.2,*/*;q=0.01';
+    // GLTF_SPEC: Figure out correct mime types (https://github.com/KhronosGroup/glTF/issues/412 and https://github.com/KhronosGroup/glTF/issues/943)
```

Comment:
Sorry, last comment.  Is this comment still needed?  1.0 and 2.0 specs are done and all mime types are known, right?